types/node/test/events.ts

```diff
@@ -111,38 +111,3 @@ async function test() {
         events.EventEmitter.captureRejectionSymbol;
     captureRejectionSymbol2 = events.captureRejectionSymbol;
 }
-
-{
```

Comment:
This was removed for now as these APIs are not actually in the node typings yet.